src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -193,6 +194,7 @@ def save_pretrained(
         variant: Optional[str] = None,
         max_shard_size: Optional[Union[int, str]] = None,
         push_to_hub: bool = False,
+        dduf_filename: Optional[Union[str, os.PathLike]] = None,
```

Comment:
```suggestion
        dduf_file: Optional[Union[str, os.PathLike]] = None,
```

I would rename it `dduf_file` for consistency within `diffusers` and with `transformers`'s `.from_pretrained(model_id, gguf_file=filename)`